<?xml version="1.0"?>
<launch>
  <arg name="model" default="$(find inverse_kinematics)/urdf/simple_arm.urdf" />
  <arg name="gui" default="false"/>

  <param name="robot_description" textfile="$(arg model)"/>
  <param name="use_gui" value="$(arg gui)"/>

  <!--node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/-->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>


  <node name="rviz" pkg="rviz" type="rviz" />
  <node name="inverse_joint_publisher" pkg="inverse_kinematics" type="inverse_joint_publisher" args="-d $(find inverse_kinematics)/launch/rviz.rviz"/>
</launch>
